pyaedt/maxwell.py

```diff
@@ -1951,16 +1951,12 @@ def create_setup(self, setupname="MySetupAuto", setuptype=None, **kwargs):
         corresponding to the native AEDT API.  The list of
         keywords here is not exhaustive.
 
-        .. note::
-           This method overrides the ``Analysis.setup()`` method for the HFSS app.
-
         Parameters
         ----------
         setuptype : int, str, optional
-            Type of the setup. Based on the solution type, options are
-            ``"HFSSDrivenAuto"``, ``"HFSSDrivenDefault"``,
-            ``"HFSSEigen"``, ``"HFSSTransient"`` and ``"HFSSSBR"``.
-            The default is ``"HFSSDrivenAuto"``.
+            Type of the setup. Based on the solution type, options are:-
```

Comment:
```suggestion
            Type of the setup. Based on the solution type, options are:
```

Reply:
I suppose the dash isn't needed. If it is of interest, ignore my remark (and please explain me its uses) :)